models/inventory/tests/testInventory.spec.ts

```diff
@@ -90,6 +119,55 @@ test('create stock movement, material receipt', async (t) => {
   t.equal(await fyo.db.getStockQuantity(itemMap.Ink.name), quantity);
 });
 
+test('Batch Enabled : create stock movement, material receipt', async (t) => {
+  const { rate } = itemMap.Pen;
+  const quantity = 2;
+  const batchNumber = batchNumberMap.batchNumberOne.name;
+  const amount = rate * quantity;
+  const stockMovement = await getStockMovement(
+    MovementType.MaterialReceipt,
+    new Date('2022-11-03T09:57:04.528'),
+    [
+      {
+        item: itemMap.Pen.name,
+        to: locationMap.LocationOne,
+        quantity,
+        batchNumber,
+        rate,
+      },
+    ],
+    fyo
+  );
+
+  await (await stockMovement.sync()).submit();
+  t.ok(stockMovement.name?.startsWith('SMOV-'));
```

Comment:
It's alright to copy existing tests, but what's the point of re-testing things that have already been added?

Batch Tests should not be concerned with Stock Movement naming.